Mathlib/LinearAlgebra/Matrix/SpecialLinearGroup.lean

```diff
@@ -481,7 +481,7 @@ open MatrixGroups
 
 open Matrix Matrix.SpecialLinearGroup
 
-local notation:1024 "↑ₘ" A:1024 => ((A : SL(2, ℤ)) : Matrix (Fin 2) (Fin 2) ℤ)
+local notation:1024 "↑ₘ" A:1024 => ((A : SL(2, ℤ)) : Mat[2,2][ℤ])
```

Comment:
only place where I'm unsure whether it matters to keep to original term. I guess it's fine?